src/draft-wullink-rpp-core.md

```diff
@@ -147,21 +147,45 @@ Some EPP result codes, like 01500, 02500, 02501 and 02502 are related to session
 
 When an error occurs that prevents processing of the requested action, an RPP server MUST respond using a Problem Detail
 document [@!RFC9457] detailing what went wrong, or what was not acceptable to the server.
-The `type` field SHOULD be a URN under the `urn:ietf:params:rpp:code:` namespace, using the RPP response code; e.g.
-with a header of `RPP-Code: 02005` the `type` field is `urn:ietf:params:rpp:code:02005`.
+The `type` field MUST be the `urn:ietf:params:rpp:problem` URN.
```

Comment:
Is there any registration procedure for urns?
If yes, a todo would be needed to add IANA text.

Reply:
BTW: error code URLs which actually show some content about an error are quite an interesting practice I saw in the wild